opds.py

```diff
@@ -1,74 +1,49 @@
-
-from collections import (
-    defaultdict,
-)
-
-from urlparse import urlparse, urljoin
-import copy
 import datetime
-import feedparser
 import logging
-import os
-import random
-import re
-import site
-import sys
-import time
 import urllib
-
-from nose.tools import set_trace
-
-from sqlalchemy.orm.query import Query
-from sqlalchemy.sql.expression import func
-from sqlalchemy.orm.session import Session
-
-import requests
+from collections import (
+    defaultdict,
+)
 
 from lxml import etree
+from sqlalchemy.orm.session import Session
 
 from cdn import cdnify
-from config import Configuration
 from classifier import Classifier
+from core.model import ExternalIntegration
```

Comment:
This import style only works when the file is being imported as part of a submodule called `core`. This causes the import problem you see in the tests. Import this class using the 'from model import' sequence a little lower in the file.

Reply:
Haven't noticed, fixed it, thank you!